jest.config.ts

```diff
@@ -21,4 +21,7 @@ module.exports = {
 	globalSetup: '<rootDir>/jest-global-setup.ts',
 	setupFilesAfterEnv: [ '<rootDir>/jest-setup.ts' ],
 	watchPlugins: [ 'jest-watch-typeahead/filename', 'jest-watch-typeahead/testname' ],
+	moduleNameMapper: {
+		'^react-markdown$': '<rootDir>/src/__mocks__/react-markdown.tsx',
+	},
```

Comment:
`react-markdown` is [ESM only](https://github.com/remarkjs/react-markdown?tab=readme-ov-file#install) and needs to be mocked for jest.

Reply:
From my quick testing, this configuration is not necessary. The mock is automatically loaded given the mock file is placed in the expected location — a `__mocks__` directory within a project `roots` directory. 

> If the module you are mocking is a Node module (e.g.: `lodash`), the mock should be placed in the `__mocks__` directory adjacent to `node_modules` (unless you configured [`roots`](https://jestjs.io/docs/configuration#roots-arraystring) to point to a folder other than the project root) and will be automatically mocked.
> — [Jest documentation](https://jestjs.io/docs/manual-mocks#mocking-node-modules)